frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-details-card.tsx

```diff
@@ -77,6 +91,15 @@ export const VMDetailsCard: React.FC<VMDetailsCardProps> = () => {
           >
             {launcherPod && ipAddrs}
           </DetailItem>
+          <DetailItem title="Operating System" error={!operatingSystem} isLoading={!vmiLike}>
+            {operatingSystem}
+          </DetailItem>
+          <DetailItem title="Time Zone" error={!timeZone} isLoading={!vmiLike}>
+            {timeZone}
+          </DetailItem>
+          <DetailItem title="Logged In Users" error={!numLoggedInUsers} isLoading={!vmiLike}>
+            {numLoggedInUsers && `${numLoggedInUsers} users`}
```

Comment:
can we do "one user" for single user ?

Reply:
wow nice ! 
we can get the "no users" string to constant and use it here too: 
https://github.com/openshift/console/blob/7cb65b6cbc5c90a786db410737636673ba395e1e/frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx#L106

Reply:
Ah, excellent points. I should be differentiating between GA data being available with no users logged in and GA data not being available. I should also be handling single vs multiple users. I'll take care of these. Thanks!

Reply:
Done!